src/main/java/com/github/dockerjava/api/model/Node.java

```diff
@@ -8,20 +9,30 @@
  * A node as returned by the /events API, for instance, when Swarm is used.
  */
 @JsonInclude(Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
 public class Node {
 
     @JsonProperty("Name")
     private String name;
 
-    @JsonProperty("Id")
+    @JsonProperty("ID")
```

Comment:
hi @KostyaSha those docs seem out of date 

https://github.com/docker/swarm/blob/master/scheduler/node/node.go#L11-L12 

the case did indeed seems to have been changed by [this commit](https://github.com/docker/swarm/commit/1297a4cef2813839da91bf962beda2e3a35cf77e#diff-a28a4b7a97d2368507318a8565e27ff2L41), like you suspected.

probably a good idea to change Node class to have both fields and return the one that is not null. what do you think?